src/cuda/spreadinterp.cpp

```diff
@@ -26,7 +26,7 @@ int setup_spreader(finufft_spread_opts &opts, T eps, T upsampfac, int kerevalmet
               __func__, upsampfac);
       return FINUFFT_ERR_HORNER_WRONG_BETA;
     }
-    if (upsampfac <= 1.0) {
+    if (upsampfac < 1.0) {
```

Comment:
Need comments here to explain that upsampfac=1.0 is valid for spreadinterponly.   Also fix error msg to <1.0 not <=1.0.